db/migrate/20230503142513_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
```

Comment:
- [ ] Your table should have a reference column that refers to the author in the user table, this should create `author_id` and a corresponding index. It can be similar to 👉 `t.references :author, foreign_key: { to_table: 'users' }` 🙃 🙃

Reply:
- [ ] It would be better if we address this part of the comment from the previous review. As you improve the Like migration table so should you do to the Posts and Comments as said earlier